tools/cargo_bazel/src/config.rs

```diff
@@ -173,6 +173,10 @@ pub struct CrateExtras {
     /// For git sourced crates, this is a the
     /// [git_repository::shallow_since](https://docs.bazel.build/versions/main/repo/git.html#new_git_repository-shallow_since) attribute.
     pub shallow_since: Option<String>,
+
+    /// Prefix between workspace root (e.g. git repository) and crate root (i.e. directory
+    /// containing Cargo.toml) which needs to be stripped in the repository rule.
+    pub strip_prefix: Option<String>,
```

Comment:
This should also be paired with change to [deps.bzl](https://github.com/abrisco/cargo-bazel/blob/0.0.13/defs.bzl#L264-L284) so users can actually set this value in the `crates_repository` rule.

Reply:
Instead I just removed this from this struct - I don't think it makes sense for users to explicitly specify it.